### PR TITLE
External link response check

### PR DESCRIPTION
**Pull request by skumarqa2010**
External link response check

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -506,3 +506,18 @@ Cypress.Commands.add("verifyErrorSummaryTitle", () => {
 Cypress.Commands.add("verifyErrorMessage", (errorMessage) => {
 	cy.get("li > a").should("exist").should("have.text", errorMessage);
 });
+
+Cypress.Commands.add("verifyExternalLinkResponse", (externalLinkText) => {
+	cy.contains("a", externalLinkText)
+		.invoke("attr", "href")
+		.then(function (val) {
+			cy.request({
+				url: val,
```
Comment (ethax-ross) [inline cypress/support/commands.js:515]: Are we not calling out to an external service here? we should avoid that or if that service becomes unavailable we won't be able to deploy

Comment (skumarqa2010) [reply cypress/support/commands.js]: Resolved

File: cypress/integration/teacher-training-adviser.spec.js [hunk 2]
```diff
@@ -2061,25 +2061,22 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
 });
 
 describe("Hyperlink navigation check : Tests execution date and time : " + new Date(), () => {
-	it('Links through to "Scotland" page', function () {
+	it('Verify "Scotland" link', function () {
 		cy.goToUrl("");
-		cy.contains("a", "Scotland").invoke("removeAttr", "target").click();
-		cy.url().then((url) => {
-			expect(url).equal(Navlinks.teachInScotland);
+		cy.contains("a", "Scotland").should((link) => {
```
Comment (ethax-ross) [inline cypress/integration/teacher-training-adviser.spec.js:2066]: Not a huge deal but these tests would run quicker if they were combined, so:

```
it("verify UK links", () => {
  cy.goToUrl("/");
  cy.contains("a", "Scotland")...
  cy.contains("a", "Wales")...
  cy.contains("a", "Nothern Ireland")...
});
```

At the minute we're needlessly reloading the page for each one

**Review by ethax-ross — APPROVED** [review]

File: cypress/integration/teacher-training-adviser.spec.js (hunk 2 above)
Comment (skumarqa2010) [reply cypress/integration/teacher-training-adviser.spec.js:2066]: Sure, I'll change it accordingly.  Thanks.

**Pull request merged**